.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2.1
+
+jobs:
+  placeholder:
+    docker:
+      - image: cimg/node:22.15-browsers
+    steps:
+      - run:
+          name: Placeholder Job
+          command: echo "This is a temporary placeholder job, to be removed once Circle Ci is not used in RC and master."
+
+workflows:
+  placeholder_workflow:
+    jobs:
+      - placeholder
```

Comment:
I had to add a minimal config, as it was failing otherwise

![Screenshot from 2025-05-29 19-29-27](https://github.com/user-attachments/assets/b5cb6045-46cc-4d8f-b0a0-49b4d00f3869)